src/f0cal/farm/client/__codegen__/cli.jj2

```diff
@@ -67,7 +67,7 @@ _cli_{{ noun }}_{{ verb }}
 
 {% macro CALLABLE(noun, verb, instance=False, requires_api_key=True, remote=False) %}
   {% set argsetter = ARGSETTER_NAME(noun, verb) %} 
-@f0cal.entrypoint(["farm", "{{ FORMAT_CLI_NOUN(noun) }}", "{{ verb }}"], args={{ argsetter }})
+@f0cal.core.entrypoint(["farm", "{{ FORMAT_CLI_NOUN(noun) }}", "{{ verb }}"], args={{ argsetter }})
```

Comment:
I am surprised that this is working. This change makes sense but I don't see the updated import lower in the file

Reply:
Ahh its because there is a * import down lower but it would still be good to explicitly change the import from f0cal to f0cal.core